tools/cli/src/mentat_cli/lib.rs

```diff
@@ -42,6 +42,8 @@ pub fn run() -> i32 {
 
     opts.optopt("d", "", "The path to a database to open", "DATABASE");
     opts.optflag("h", "help", "Print this help message and exit");
+    opts.optopt("q", "query", "Execute a query on startup. Queries are executed after any transacts.", "QUERY");
```

Comment:
Is this better than having folks pipe from a file or heredoc?  Is there a reason to not accept `-t ... -q ...` in any order and just evaluate them in order?  I see that you're building a set of initial commands without regard to the type of the command anyway.